openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -42,15 +42,17 @@ def __init__(self, ocp_cluster: Dict[str, Any], user_input: UserInput) -> None:
             registry_config_file=self.user_input.registry_config_file,
             docker_config_file=self.user_input.docker_config_file,
         )
+        self.fips = self.cluster_info.get("fips")
```

Comment:
no need to set this in a destroy cluster flow.
```suggestion
        if if self.user_input.create and (self.fips := self.fips = self.cluster_info.get("fips")):
```

Reply:
if `if self.user_input.create` is False then the second part of the `and` will not execute, and `self.fips` will not be set at all.
We have to have `self.fips` even if it's empty (`None`) since it's required in [_ipi_download_installer](https://github.com/RedHatQE/openshift-cli-installer/blob/main/openshift_cli_installer/libs/clusters/ipi_cluster.py#L70) (triggered [here](https://github.com/RedHatQE/openshift-cli-installer/blob/main/openshift_cli_installer/libs/clusters/ipi_cluster.py#L46) on destroy).

Reply:
```
>>> x={"a":1}
>>> x["a"]==2 and (fips := x.get("fips"))
False
>>> fips
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'fips' is not defined
```

Reply:
```
   else:
       self.fips = False
```

Reply:
Using `self.cluster_info.get("fips")` still ignores the option `fips=false` set in cluster configs. Please see my last comment in the thread with Meni above.